app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Interfaces/APICallerInterface.java

```diff
@@ -0,0 +1,22 @@
+package edu.grinnell.appdev.grinnelldirectory.Interfaces;
+
+import java.util.List;
+
+import edu.grinnell.appdev.grinnelldirectory.Model.Person;
+
+/**
+ * Created by nicholasroberson on 2/15/17.
+ */
+
+public interface APICallerInterface {
+
+    List<Person> simpleSearchCallSuccess(List<Person> people);
+
+    List<Person> advancedSearchCallSuccess(List<Person> people);
```

Comment:
Let's change this so that we only have one `onSearchSuccess` method instead of two for simple and advanced, since we can only display information from one of the search methods at once